__tests__/__unit__/extension.unit.test.ts

```diff
@@ -1375,907 +1494,769 @@ describe("Extension Unit Tests", () => {
-            isDirty: null,
-            isClosed: null,
-            save: null,
-            eol: null,
-            lineCount: null,
-            lineAt: null,
-            offsetAt: null,
-            positionAt: null,
-            getText: null,
-            getWordRangeAtPosition: null,
-            validateRange: null,
-            validatePosition: null
-        };
+    it("Testing that that openPS credentials prompt works with favorites", async () => {
+        showTextDocument.mockReset();
+        openTextDocument.mockReset();
+        showQuickPick.mockReset();
+        showInputBox.mockReset();
+        const sessionwocred = new imperative.Session({
+            user: "",
+            password: "",
+            hostname: "fake",
+            port: 443,
+            protocol: "https",
+            type: "basic",
+        });
+        const dsNode = new ZoweDatasetNode("[test]: TEST.JCL", vscode.TreeItemCollapsibleState.Expanded, sessNode, sessionwocred);
+        dsNode.contextValue = globals.DS_PDS_CONTEXT + globals.FAV_SUFFIX;
+        Object.defineProperty(Profiles, "getInstance", {
+            value: jest.fn(() => {
+                return {
+                    allProfiles: [{name: "firstName", profile: {user:undefined, password: undefined}}, {name: "secondName"}],
+                    defaultProfile: {name: "firstName"},
+                    promptCredentials: jest.fn(()=> {
+                        return ["fake", "fake", "fake"];
+                    }),
+                    loadNamedProfile: mockLoadNamedProfile,
+                    validProfile: ValidProfileEnum.VALID,
+                    checkCurrentProfile: jest.fn(),
+                };
+            })
+        });
 
-        const testDocLowercase: vscode.TextDocument = {
-            fileName: path.join(globals.DS_DIR, "/sestest/hlq.test.lowercase"),
-            uri: null,
-            isUntitled: null,
-            languageId: null,
-            version: null,
-            isDirty: null,
-            isClosed: null,
-            save: null,
-            eol: null,
-            lineCount: null,
-            lineAt: null,
-            offsetAt: null,
-            positionAt: null,
-            getText: null,
-            getWordRangeAtPosition: null,
-            validateRange: null,
-            validatePosition: null
-        };
+        showInputBox.mockReturnValueOnce("fake");
+        showInputBox.mockReturnValueOnce("fake");
 
-        const testResponse = {
-            success: true,
-            commandResponse: "",
-            apiResponse: {
-                items: []
-            }
-        };
+        await dsActions.openPS(dsNode, true, testTree);
+        expect(openTextDocument.mock.calls.length).toBe(1);
+        expect(showTextDocument.mock.calls.length).toBe(1);
+    });
 
-        // If session node is not defined, it should take the session from Profile
+    it("Testing that that openPS credentials prompt ends in error", async () => {
```

Comment:
All credential checks have been covered in profiles.unit.test.ts so they can be removed